api/admin.py

```diff
@@ -3,13 +3,18 @@
 
 from django.contrib import admin
 from .models import CustomizationRequest
+from wagtail.core.models import Page
 
 class CustomizationRequestResource(resources.ModelResource):
     class Meta:
         model = CustomizationRequest
         fields = ('id', 'email', 'created', 'complete', 'num_students', 'book', 'reason', 'modules')
```

Comment:
Does it work if you do `book__title` here?

Reply:
If not, I would suggest using the `ForeignKeyWidget` instead. Which I think would like something like this:

```
class CustomizationRequestResource(resources.ModelResource):
	book = fields.Field(
        column_name='book',
        attribute='book',
        widget=ForeignKeyWidget(Book, 'title'))

    class Meta:
        model = CustomizationRequest
        fields = ('id', 'email', 'created', 'complete', 'num_students', 'book', 'reason', 'modules')
```

Reply:
here's the documentation for that: https://django-import-export.readthedocs.io/en/latest/api_widgets.html#import_export.widgets.ForeignKeyWidget

Reply:
Modified to use widget